.github/workflows/release.yml

```diff
@@ -170,7 +173,7 @@ jobs:
           - project_name: Sso
           - project_name: Web
             release_tag: ${{ needs.setup.outputs._WEB_RELEASE_TAG }}
-          - project_name: Scim
+          - project_name: Web-sh
```

Comment:
We don't currently publish `web-sh` on Docker Hub.  We still need to publish `scim`.  I think this needs to be reverted.  Please put `scim` in the list where it should go alphabetically.

Reply:
I figured that `Web-sh:latest` is released in `clients` so I removed it.